inductiva/tasks/streams.py

```diff
@@ -60,18 +60,23 @@ class TaskStreamConsumer:
     PING_TIMEOUT_SEC = 5.
     TICK_INTERVAL_SEC = 1.
     END_OF_STREAM = "<<end_of_stream>>"
+    ALLOWED_IO_STREAMS = ["std_out", "std_err"]
 
     def __init__(self,
                  task_id: str,
                  fout: IO = sys.stdout,
-                 ferr: IO = sys.stderr):
+                 ferr: IO = sys.stderr,
+                 io_stream: str = None,
+                 no_color: bool = False):
```

Comment:
Should add the `no_color` on the docstring as well

Reply:
Done